ProjectSidewalk/SidewalkWebpage
# 2633 City Dropdown in Gallery

Resolves #2633 

I have implemented a city "filter" to the gallery page such that when a new city is selected, the page redirects you to the gallery page of the new city's server. The label filter should also persist (I have tried to show this by console logging the redirect link and opening the same link in local host as seen in the video attached). Note that the back button does not work as expected, but I will try to fix this issue in another branch. Similarly, the severities also do not persist. 

Additionally, I have added a status field to the cityparams, so that only public cities (and the current city you are in) show up as options in the city dropdown). 

##### Before/After screenshots (if applicable)
I am not sure if a before screen shot is applicable, but here is a video of the feature in action:

https://user-images.githubusercontent.com/15069663/154777805-8a10e482-4c8d-4221-ab51-b89d40cbc6e7.mp4



##### Testing instructions
1. Open Gallery and it should atomatically show which city you are in
2. Select a new city, and you should be redirected to a new server and show your new city in the city filter (note that I was unsure how to test this fully as the new server does not have my code implemented on it to show a city filter)
3. Select a filter and then select a new city, and the filter should persist.
